chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -2,7 +2,7 @@
 
 cookbook_file "#{node[:cache_location]}#{node[:openstack][:nova][:file]}" do
   source node[:openstack][:nova][:file]
-  not_if { ::File.exists?(node[:openstack][:nova][:installed]) }
+  not_if { ::File.exists?(node[:openstack][:nova][:file]) }
```

Comment:
Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.